REQUIREMENTS.md

```diff
@@ -1,4 +1,4 @@
-Depending on the [platform](https://monogame.net/articles/platforms/) that you are targeting, MonoGame has different sets of requirements.
+Depending on the [platform](https://monogame.net/articles/platforms.html) that you are targeting, MonoGame has different sets of requirements.
```

Comment:
Thanks for the contribution. I think this was fixed with a recent website update the non html url is now working.

Reply:
Hey @dellis1972 when clicking the link without the .html and the / as it currently is you're taken to a 404. Removing the .html does work, but not with the current / at the end. I can remove the .html as well as the / and it will fix the current issue and correctly route with the recent changes. 

But it is currently still a broken link without this change.

Reply:
@AristurtleDev any thoughts on this one?

Reply:
I would be in favour of just removing the `/` unless other links use `.html`. We should be consistent at least.

Reply:
@dellis1972 the initial port of the website to 11ty did in fact use the 'prettier" urls that end with `/`.  However, after implementing it this way, it was discovered that this broke all search engine results and other existing articles/documentation that point to the website, so it was reverted back during the maintainability update in PR https://github.com/MonoGame/monogame.github.io/pull/91

So @JakeLegendXIII has it correct here that it should end with `.html` and not `/`